sdk/include/game-contract-sdk/game_base.hpp

```diff
@@ -348,14 +357,19 @@ class game : public eosio::contract {
             return;
         }
 
-        eosio::check(quantity.symbol == core_symbol, "invalid token symbol");
-
+        // token verifying
+        const auto platform = get_platform();
```

Comment:
it's not a mistake but I think we could get rid of `platform` variable

Reply:
also why is `get_token_contract` check not enough?

Reply:
hmm, it's enough